test/controllers/search_controller_test.rb

```diff
@@ -90,6 +90,31 @@ def mock_timdex_search_success
     end
   end
 
+  test 'index shows basic search form when GDT is disabled' do
+    # GDT is disabled by default in test setup
+    get '/'
+    assert_response :success
+
+    # Should show basic form without geo elements
+    assert_select 'form#search-form', { count: 1 }
```

Comment:
Is it worth having an assertion to confirm `#search-form-geo` isn't present, to keep the reverse pattern from below?
```
assert_select 'form#search-form-geo', { count: 0 }
```